code/game/objects/items/stacks/medical.dm

```diff
@@ -9,6 +9,195 @@
 	var/heal_brute = 0
 	var/heal_burn = 0
 
+/obj/item/stack/medical/proc/radial_medical(mob/living/carbon/M as mob, mob/user as mob)
```

Comment:
```suggestion
/obj/item/stack/medical/proc/radial_medical(mob/living/carbon/patient, mob/doctor)
```
1) Informative vars.
2) `as type` is only useful for verbs and stuff where the person needs to select from lists and the likes.